tests/test_circuit.py

```diff
@@ -1186,6 +1186,34 @@ def cost_fn(_params):
             assert not np.isnan(grad[ansatz3_param])
 
 
+@pytest.mark.parametrize("backend", [lf("npb"), lf("tfb"), lf("jaxb")])
+def test_qiskit_vs_tc_intialization(backend):
+    try:
+        from qiskit import QuantumCircuit
+        import qiskit.quantum_info as qi
+    except ImportError:
+        pytest.skip("qiskit is not installed")
+
+    n = 3
+
+    qis_c = QuantumCircuit(n)
+    qis_c.h(0)
+    qis_c.cnot(0, 1)
+    qis_c.y(2)
+    state = qi.Statevector(qis_c)
+    qis_c = QuantumCircuit(n)
+    qis_c.initialize(state)
+    c = tc.Circuit.from_qiskit(qis_c)
```

Comment:
I suggest add a gate here to make the test more universal (not only initialize instruction) ``c.cx(1, 2)`` and also ``c2.cx(1, 2)`` accordingly